fuse-prometheus-operator.yml

```diff
@@ -0,0 +1,328 @@
+  spec:
+    group: monitoring.coreos.com
+    names:
+      kind: Alertmanager
+      listKind: AlertmanagerList
+      plural: alertmanagers
+      singular: alertmanager
+    scope: Namespaced
+    version: v1
+
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: prometheus-operator
+    namespace: ${NAMESPACE}
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: Role
+    name: prometheus-operator
+  subjects:
+  - kind: ServiceAccount
+    name: prometheus-operator
+    namespace: ${NAMESPACE}
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
+  metadata:
+    name: prometheus-operator
+  rules:
+  - apiGroups:
+    - apiextensions.k8s.io
+    resources:
+    - customresourcedefinitions
+    verbs:
+    - '*'
+  - apiGroups:
+    - monitoring.coreos.com
+    resources:
+    - alertmanagers
+    - prometheuses
+    - prometheuses/finalizers
+    - alertmanagers/finalizers
+    - servicemonitors
+    - prometheusrules
+    verbs:
+    - '*'
+  - apiGroups:
+    - apps
+    resources:
+    - statefulsets
+    verbs:
+    - '*'
+  - apiGroups:
+    - ""
+    resources:
+    - configmaps
+    - secrets
+    verbs:
+    - '*'
+  - apiGroups:
+    - ""
+    resources:
+    - pods
+    verbs:
+    - list
+    - delete
+  - apiGroups:
+    - ""
+    resources:
+    - services
+    - endpoints
+    verbs:
+    - get
+    - create
+    - update
+  - apiGroups:
+    - ""
+    resources:
+    - nodes
+    verbs:
+    - list
+    - watch
+  - apiGroups:
+    - ""
+    resources:
+    - namespaces
+    verbs:
+    - list
+    - watch
+
+- apiVersion: apps/v1beta2
+  kind: Deployment
+  metadata:
+    labels:
+      k8s-app: prometheus-operator
+    name: prometheus-operator
+    namespace: ${NAMESPACE}
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        k8s-app: prometheus-operator
+    template:
+      metadata:
+        labels:
+          k8s-app: prometheus-operator
+      spec:
+        containers:
+        - args:
+          - --manage-crds=false
+          - --namespace=${NAMESPACE}
+          - --logtostderr=true
+          - --config-reloader-image=quay.io/coreos/configmap-reload:v0.0.1
+          - --prometheus-config-reloader=quay.io/coreos/prometheus-config-reloader:v0.23.2
+          image: quay.io/coreos/prometheus-operator:v0.23.2
+          name: prometheus-operator
+          ports:
+          - containerPort: ${CONTAINER_PORT}
+            name: http
+          resources:
+            limits:
+              cpu: ${CPU_LIMIT}
+              memory: ${MEMORY_LIMIT}
+            requests:
+              cpu: ${CPU_REQUEST}
+              memory: ${MEMORY_REQUEST}
+        serviceAccountName: prometheus-operator
+
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: prometheus-operator
+    namespace: ${NAMESPACE}
+
+- apiVersion: monitoring.coreos.com/v1
+  kind: Prometheus
+  metadata:
+    name: prometheus
+    namespace: ${NAMESPACE}
+  spec:
```

Comment:
I had to add the following attribute to the resource otherwise the Prometheus pod couldn't be created:

```yaml
    securityContext: {}
```

Reply:
Added